packages/components-react/code-block-react/src/code-block.tsx

```diff
@@ -2,13 +2,27 @@ import type { HTMLAttributes } from 'react';
 import { clsx } from 'clsx';
 import { forwardRef } from 'react';
 
-export type CodeBlockProps = HTMLAttributes<HTMLPreElement>;
+export interface CodeBlockProps extends HTMLAttributes<HTMLPreElement> {
+  line?: 'wrap' | 'nowrap' | 'overflow';
+}
 
 export const CodeBlock = forwardRef<HTMLPreElement, CodeBlockProps>(function CodeBlock(props, forwardedRef) {
-  const { children, className, ...restProps } = props;
+  const { children, className, line = 'wrap', ...restProps } = props;
 
   return (
-    <pre dir="ltr" translate="no" {...restProps} className={clsx('nl-code-block', className)} ref={forwardedRef}>
+    <pre
+      dir="ltr"
+      translate="no"
+      tabIndex={line === 'overflow' ? 0 : undefined}
+      {...restProps}
+      className={clsx(
+        'nl-code-block',
+        line === 'nowrap' && 'nl-code-block--nowrap',
+        line === 'overflow' && 'nl-code-block--overflow',
+        className,
+      )}
```

Comment:
```suggestion
      className={clsx(
        'nl-code-block',
        {
          ['nl-code-block--nowrap']: line === 'no-wrap',
          ['nl-code-block--overflow']: line === 'overflow',
        },
        className,
      )}
```

Voor leesbaarheid en consistentie?